kpi/management/commands/populate_kc_xform_kpi_asset_uid.py

```diff
@@ -76,10 +76,11 @@ def handle(self, *args, **options):
             try:
                 if asset.deployment.set_asset_uid(force=force):
                     if verbosity >= 2:
-                        self.stdout.write('\nAsset #{}: Patching XForm'.format(asset.id))
+                        self.stdout.write(
+                            '\nAsset #{}: Patching XForm'.format(asset.id))
                     # Avoid `Asset.save()` logic. Do not touch `modified_date`
                     Asset.objects.filter(pk=asset.id).update(
-                        _deployment_data=asset._deployment_data)
+                        _deployment_data=asset.deployment.get_data())
```

Comment:
opportunity to use `save_to_db()`?

Reply:
no, because we want to completely overwrite the destination `_deployment_data`, not merge into it (using `ReplaceValues`)